pkg/component/controller/etcd_member_reconciler.go

```diff
@@ -82,8 +122,9 @@ func (e *EtcdMemberReconciler) Start(ctx context.Context) error {
 		// Create the object for this node
 		err = e.createMemberObject(ctx)
 		if err != nil {
-			log.WithError(err).Error("failed to create EtcdMember object")
+			log.WithError(err).Error("failed to create EtcdMember object for myself")
```

Comment:
Software with an ego ... :upside_down_face:

What about "for this controller"?